### DISCUSSION TITLE
How Skywalking monitors the CPU, memory, and disk usage of servers and uses alarm functions to alert

### DISCUSSION THREAD

**Discussion by YoYoxiaoai** [apache/skywalking#11777] [Q&A] [answered]
env:
oap:9.5.0   skywalking-agent:8.15.0
I want to know the oal rules to alert CPU usage when it's too high

**Comment by wu-sheng** [comment, marked as answer]
Please read the alerting rules, it is about metrics only. No matter it is collected by OAL or MAL. 
You could make your UI into edit mode, and you are able to see the metrics name(or expression)

**Reply by YoYoxiaoai** [reply]
did the alarms-setting.yaml collected by MAL? could I write the OAL with metrics name?

**Reply by YoYoxiaoai** [reply]
alarms-setting.yml  --->
![image](https://github.com/apache/skywalking/assets/56816886/0bc7bdce-52dc-4e99-aeb1-a58e15e86aed)
vm.yaml  --->
![image](https://github.com/apache/skywalking/assets/56816886/675478ed-8fb3-4616-a866-33e10010bc60)

it's not work

**Reply by wu-sheng** [reply]
OAL is not as same as MAL metrics. But you can use MAL metrics. You could refer to the UI about the name.
